### PR TITLE
fix(common/core/web): mnemonic modifier key-up handling

### LINKED ISSUE #3744: bug(web): desktop osk is not showing modifier keys released when they are released on physical keyboard
**Describe the bug**
On desktop KeymanWeb, when I press the Shift key on my hardware keyboard (e.g. on keymanweb.com), the OSK shows the Shift layer. When I release the Shift key, the Shift layer remains active. This does not happen on all keyboards but does happen on SIL Philippines. (For example, it doesn't appear to happen on Khmer Angkor).

**To Reproduce**
1. Go to https://keymanweb.com/#tl,Keyboard_sil_philippines (Tagalog language)
2. Make sure the OSK is active.
3. Press and release shift

image

**Expected behavior**
When I release the Shift key on the hardware keyboard, the OSK should refresh to the default layer.

---

**KeymanWeb:**
 - OS: Windows 10
 - Browser: Chrome, Firefox
 - KeymanWeb Version: 13.0.109
 - URL: https://keymanweb.com/#tl,Keyboard_sil_philippines
  
---
 
**Keyboard**
 - Keyboard name: sil_philippines
 - Language name: Tagalog

### PR DESCRIPTION
**Pull request by jahorton**
Closes #3744.

Turns out `sil_philippines` is a mnemonic keyboard; related code paths were at fault.  While key-down events still triggered OSK changes, this was because of this block:

https://github.com/keymanapp/keyman/blob/e445e3d034162ab525c9adc9f696b391ed8bda96/common/core/web/keyboard-processor/src/text/keyboardProcessor.ts#L662-L665

Said block _immediately_ follows the adjusted, previously-bug-affected section and does _not_ need a backing `Lkc.Lcode` value.

### CODE REVIEW

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.219-beta](https://keyman.com/downloads/releases/beta/14.0.219)

File: common/core/web/keyboard-processor/src/text/keyboardProcessor.ts [hunk 1]
```diff
@@ -618,6 +623,20 @@ namespace com.keyman.text {
...
     }
 
+    static isModifier(Levent: KeyEvent): boolean {
+      switch(Levent.Lcode) {
+        case 16: //"K_SHIFT":16,"K_CONTROL":17,"K_ALT":18
+        case 17: 
+        case 18: 
+        case 20: //"K_CAPS":20, "K_NUMLOCK":144,"K_SCROLL":145
+        case 144:
+        case 145:
```
Comment (mcdurdin) [inline common/core/web/keyboard-processor/src/text/keyboardProcessor.ts:633]: I would have preferred this to have the comment for each one on each line...

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.9-alpha](https://keyman.com/downloads/releases/alpha/15.0.9)

**Pull request merged**